cmd/titus-storage/mount.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+const (
+	mountCommand = "/apps/titus-executor/bin/titus-mount-block-device"
+)
+
+func calculateFlags(mountPerm string) string {
+	if mountPerm == "RW" {
+		return "1"
+	} else if mountPerm == "RO" {
+		return "0"
+	}
+	log.Fatalf("error parsing the mount permissions: '%s', needs to be only RW/RO", mountPerm)
+	return "0"
+}
+
+func mountIt(dev string, fstype string, mountPoint string, mountPerm string) error {
+	flags := calculateFlags(mountPerm)
+	_ = os.Chdir("/tmp")
+	pid1 := os.Getenv("TITUS_PID_1_DIR")
+	if pid1 == "" {
+		log.Fatalf("TITUS_PID_1_DIR is not set!")
+	}
+	log.Printf("Running %s to mount %s onto %s in the container", mountCommand, dev, mountPoint)
+	log.Printf("TITUS_PID_1_DIR=%s MOUNT_TARGET=%s MOUNT_OPTIONS=source=%s MOUNT_FLAGS=%s MOUNT_FSTYPE=%s %s", pid1, mountPoint, dev, flags, fstype, mountCommand)
```

Comment:
This log line might get out of sync, consider logging `cmd.Env`.

Reply:
great idea